saga/inMemorySagaLog.go

```diff
@@ -29,6 +31,14 @@ func MakeInMemorySagaLog() SagaLog {
 	}
 }
 
+type SagaLogMemoryConfig struct {
```

Comment:
It doesn't for now, because right now it's the only option.

It *would* need a config *if*:
a) there are multiple options
b) we want to let people switch them at runtime (based on the command-line).

Right now, neither of these is true, so I can remove it.